geniesp/bpc_redcap_export_mapping.py

```diff
@@ -1241,11 +1241,20 @@ def get_timeline_performance(
         timeline_infodf.index = timeline_infodf["code"]
         data = self.create_fixed_timeline_files(timeline_infodf, "TIMELINE-PERFORMANCE")
         # HACK: Due to remapping logic, we will re-create RESULT column with correct
-        # values
-        data['df']['MD_KARNOF'] = data['df']['RESULT']
+        has_md_karnof = ~data['df']['MD_KARNOF'].fillna('').str.startswith(("Not" ,"not"))
```

Comment:
We need to determine whether or not strings that start with "Not" is a good enough indicator for rows not having these values.

Note: already thing that we need to add blank string to this